docs/usage.md

```diff
@@ -417,3 +417,75 @@ podman build . \
   --network none \
   --tag atomic-reactor
 ```
+
+### Example: npm
```

Comment:
You can also link it in the table of contents https://github.com/lkolacek/cachi2/blob/npm-docs/docs/usage.md#usage

Reply:
Added link